tests/unit/store.spec.ts

```diff
@@ -28,7 +28,7 @@ describe('getter tests', () => {
         { name: 'rename', oldname: 'baz', newname: 'spam' },
       ];
       const state = buildState(stateWithOnePipeline(pipeline));
-      expect(getters.activePipeline(state)).toEqual(pipeline);
+      expect(getters.activePipeline(state, {}, {}, {})).toEqual(pipeline);
```

Comment:
Hmm, I did not take the time to check / dig but this doesn't feel right. We shouldn't need to pass those 3 extra empty objects.

Reply:
I'm afraid we do.
For better type inference, I added the type `GetterTree` to the `getters` object:
https://github.com/ToucanToco/weaverbird/pull/439/commits/9eedeee50f8aab95a1d333751c1833f1e56e1179
That means that every function defined there is of type `Getter`.
A `Getter` is a function that takes as arguments `ModuleState`, `ModuleGetters`, `RootState`, `RootGetters`. All are required.

If we want to avoid this (which I won't recommend), the workaround would be to define the functions alone, and then to add them into the `GetterTree getters` object later.
This way, using their "standalone" version comes with the inferred type `VQBState => something` rather that the `Getter` type.
I personally prefer the first option, where it become kinda obvious we're having a getter.

Reply:
OK fine. the `@Getters` decorator will handle extra arguments.